src/GraphQL.Tests/Types/AutoRegisteringInterfaceGraphTypeTests.cs

```diff
@@ -372,35 +361,10 @@ public async Task ThrowsWhenSourceNull()
         var graphType = new AutoRegisteringInterfaceGraphType<NullSourceFailureTest>();
         var context = new ResolveFieldContext();
         (await Should.ThrowAsync<InvalidOperationException>(async () => await graphType.Fields.Find("Example1")!.Resolver!.ResolveAsync(context).ConfigureAwait(false)).ConfigureAwait(false))
-            .Message.ShouldBe("IResolveFieldContext.Source is null; please use static methods when using an AutoRegisteringInterfaceGraphType as a root graph type or provide a root value.");
+            .Message.ShouldBe("This field resolver should never be called. It is only used to prevent the default field resolver from being used.");
         (await Should.ThrowAsync<InvalidOperationException>(async () => await graphType.Fields.Find("Example2")!.Resolver!.ResolveAsync(context).ConfigureAwait(false)).ConfigureAwait(false))
-            .Message.ShouldBe("IResolveFieldContext.Source is null; please use static methods when using an AutoRegisteringInterfaceGraphType as a root graph type or provide a root value.");
-    }
-
-#if !NET48 // .NET Framework 4.8 does not support default interface implementation, so would not support static methods on an interface
-    [Fact]
-    public async Task WorksWithNullSource()
```

Comment:
This removed test doesn't really apply anymore; it's just going to throw the same exception as ThrowsWhenSourceNull.